include/openssl/ssl.h

```diff
@@ -2445,7 +2474,7 @@ void ERR_load_SSL_strings(void);
 # define SSL_R_WRONG_SSL_VERSION                          266
 # define SSL_R_WRONG_VERSION_NUMBER                       267
 # define SSL_R_X509_LIB                                   268
-# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           269
+# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           101
```

Comment:
I have no idea about the black-magic of mkerrs.  And who cares if the value  changes compared to 1.0.2?